plugins/module_utils/job.py

```diff
@@ -374,7 +376,12 @@ def _get_job_status(job_id="*", owner="*", job_name="*", dd_name=None, dd_scan=T
             job["ret_code"]["steps"] = []
             job["ddnames"] = []
             job["duration"] = duration
-            job["execution_time"] = entry.execution_time
+            if hasattr(entry, "execution_time"):
+                job["execution_time"] = entry.execution_time
+            if hasattr(entry, "system"):
+                job["system"] = entry.system
+            if hasattr(entry, "subsystem"):
```

Comment:
In this part looks like If we call the function with dd_scan=True it would now return a different value for system and subsystem like when we use it from `job_output` function, used by `zos_job_output` module.

That would create some inconsistencies in the results from one module to another, I suggest we add a check like if we already have populated "system" or "subsystem" do not fetch it from the dd_scan log.

Reply:
This in line 488.